src/lib/spinel/spinel.h

```diff
@@ -2370,6 +2370,12 @@ enum
      */
     SPINEL_PROP_NET_PSKC = SPINEL_PROP_NET__BEGIN + 11,
 
+    /// Instruct NCP to leave the current network gracefully
+    /** Format `b` - Write only
```

Comment:
nit: I think there is no payload for this property now. We should remove the `Format b`.

Reply:
Thanks! Changed to 'Format emtpy'.